Zune.Net.Catalog.Image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# https://hub.docker.com/_/microsoft-dotnet
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+WORKDIR /source
+
+# copy csproj and restore as distinct layers
+COPY ./ .
```

Comment:
Ideally this shojld be broken into layers of dependency. I need to research the right way to do this for a .net project like this.